src/tokens/solana.tokenlist.json

```diff
@@ -11241,7 +11241,7 @@
     },
     {
       "chainId": 101,
-      "address": "5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA",
+      "address": "8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ",
```

Comment:
@bartosz-lipinski DXB contacted me and saying they're wanting to change to this new address. there shouldn't be any devastating thing from technical aspect by updating this already-published address (= mint address)? or there should be new entry for  true `DXB` and change of this entry `name` to `DefiXBet Token (obsolete; test-purpose)`?


They said all of already minted tokens (at [5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA](https://explorer.solana.com/address/5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA/largest)) was used test-purpose and it won't be circulating actually.

Reply:
We will distribute our new token with this token address 
8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ